scanner/src/test/java/edu/ucr/cs/riple/scanner/tools/MethodInfoDisplay.java

```diff
@@ -104,6 +107,9 @@ public String toString() {
         + ", hasNullableAnnotation='"
         + hasNullableAnnotation
         + '\''
```

Comment:
nit: Same as above. String.format() over concatenation.

Reply:
This one actually does feel more to me like a case where `String.format()` could make the code more readable than plain concatenation.

Reply:
These are generated by Intellij and the only purpose is to just show a hint to the user what wrong output failed the test. Is it really worth it to undo a generated code here :) ?

Reply:
Fair enough. Fine to keep the generated version, then.

Reply:
lol! This stirred up too much conversation :O